EpicLoot/src/Magic/MagicItemEffects/DecreaseForsakenCooldown.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Emit;
+using HarmonyLib;
+
+namespace EpicLoot.MagicItemEffects;
+
+public class DecreaseForsakenCooldown
+{
+    [HarmonyPatch(typeof(Player), nameof(Player.ActivateGuardianPower))]
+    private class DecreaseForsakenCooldown_Player_ActivateGuardianPower_Patch
+    {
+        private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructionsEnumerable)
```

Comment:
I can see this transpiler being very fragile if a future update changes this method. Another way to achieve this is to check if m_guardianPowerCooldown is equal to m_guardianSE.m_cooldown in the postfix then apply changes. I think either way will work, but aiming for more robust code to stand against volatile game updates might be the better path for long term maintenance.

Reply:
Finished in [commit](https://github.com/OrianaVenture/Randy_Vapok_ValheimMods/pull/10/commits/5b885e9615288a1117bd17569c590ca2bd66dd9e)